setup.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+import hashlib
+import http
+import io
+import os.path
+import stat
+import sys
+import tarfile
+import urllib.request
+import zipfile
+from distutils.command.build import build as orig_build
+from distutils.core import Command
+from typing import Tuple
+
+from setuptools import setup
+from setuptools.command.install import install as orig_install
+
+TYPOS_VERSION = '1.0.9'
```

Comment:
Please update `Cargo.toml`s `[package.metadata.release]` table to ensure we update this automatically on each release

Reply:
Gah, how did I forget about that, when I remembered and documented the limitation on sha256sums :thinking: :stuck_out_tongue: 

Hopefully fixed by ef76d20c6a8060fb1036d4b0db0d0c29d441ff82